Topics/Teamwork.md

```diff
@@ -1,5 +1,22 @@
 ## Resources for Teamwork
 
+## Communication
```

Comment:
Overall, the paragraph provides a good overview of the importance of communication and conflict resolution in software engineering teams.

Reply:
Thank you!